core/src/main/java/io/grpc/Attributes.java

```diff
@@ -29,8 +29,8 @@
 
 /**
  * An immutable type-safe container of attributes.
+ * @since 1.13.0
  */
-@ExperimentalApi("https://github.com/grpc/grpc-java/issues/1764")
```

Comment:
Since this is being removed, we need some way to denote that the `@Deprecated` things will actually be removed. You could copy this attribute to those deprecated methods, or maybe something similar.